modules/fluentd/templates/fluent.conf

```diff
@@ -69,12 +69,16 @@
     @type elasticsearch
     host ${elasticsearch_hostname}
     scheme https
-    logstash_format true
-    logstash_prefix "$${tag}"
     port ${elasticsearch_port}
     flush_interval 5s
     include_tag_key true
 
+    logstash_format true
+    logstash_prefix "$${tag}"
+    {{ if eq (keyOrDefault "${weekly_index_enabled_consul_key}" "false") "true" }}
+    logstash_prefix %xxxx.%ww
```

Comment:
https://www.joda.org/joda-time/apidocs/org/joda/time/format/DateTimeFormat.html

Reply:
Have you verified for `logstash_prefix`, if putting down a config value twice is an allowed / idiomatic way to override the value?

If not, `if`-`else`-`end` would seem like a safer bet here.